tests/helper/helper_generic.go

```diff
@@ -16,16 +17,15 @@ import (
 	"github.com/onsi/gomega/gexec"
 )
 
-func init() {
-	rand.Seed(time.Now().UTC().UnixNano())
-}
-
 // RandString returns a random string of given length
 func RandString(n int) string {
 	const letterBytes = "abcdefghijklmnopqrstuvwxyz"
 	b := make([]byte, n)
 	for i := range b {
-		b[i] = letterBytes[rand.Intn(len(letterBytes))]
+		// this error is ignored because it fails only when the 2nd arg of Int() is less then 0.
+		// which wont happen
```

Comment:
and the reason it won't happen is because the length of `letterBytes` is always 26 :tada: